code-challenges.test.js

```diff
@@ -9,51 +12,207 @@
-
-const numbersArray1 = [6, 7, 8, 9, 10]
+describe("multBy3", () => {
+  it("takes an array of numbers and returns an array with all of the numbers multiplied by 3", () => {
+    const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+    expect(multBy3(numbersArray1)).toEqual(([18, 21, 24, 27, 30]))
+    expect(multBy3(numbersArray2)).toEqual(([72, 81, 90, 99, 108]))
+  })
+})
+//  ReferenceError: multBy3 is not defined
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
+// b) Create the function that makes the test pass.
+// parameter will be an array of numbers called array
+// Iterate on array using .map method
+// take current value multiply by 3
+// return array 
+ const multBy3 = (array) => {
+    return array.map(value => value * 3)
+ }
+//  PASS  ./code-challenges.test.js
+//   multBy3
+//   ✓ takes an array of numbers and returns an array with all of the numbers multiplied by 3 (2 ms)
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+
+// --------------------1)
+
+// // a) Create a test with expect statements for each of the variables provided.
+describe("divisbleBy3", () => {
+    it("takes an object as an argument and returns with a statement on if the value inside it is evenly divisible by the number 3", () => {
+        const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
 const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+expect(divisbleBy3(object1)).toEqual("15 is divisible by three"),
+expect(divisbleBy3(object2)).toEqual ("0 is divisible by three"),
+expect(divisbleBy3(object3)).toEqual("-7 is not divisible by three")
+
+    })
+    })
+    // ReferenceError: divisbleBy3 is not defined
+
+
+
 
 // b) Create the function that makes the test pass.
+// psuedocode :create a function named divisbleBy3 that takes in an object as an argument and number as a value. 
+// input will be an object , the parameter will be named division. 
+// iterate on object via string interpolation to %3 ===0 to determine if the objects value is divisible by 3.
+// expected output  return a statment stating "15 is divisible by three","0 is divisible by three" and "-7 is not divisible by three"
+// it took me about 4 hours to complete this assessment challenge question. I believe I overcomplicated the syntax in my head and struggled to properly express what I meant to accomplish during the process. In the future I will use my psuedocode as more of a guideline to keep me on the right path. 
```

Comment:
Good job being honest about it. I think that maybe having the parameter named the "thing" that your function is taking, may have helped here too. I can see how you defined your input was going to be an object so seeing your function reading as 'object.number' would help it make a bit more sense.